spec/feature_tests/signin_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature "Sign in" do
+  scenario "Person signs in to use chitter" do
+    visit '/signin'
+    expect(page).to have_content 'Please sign in'
```

Comment:
This test is not testing their ability to sign in. It tests that the string 'Please sign in' is being shown on the page. A better test would interact with the page to give it some information to sign in. Eg click button sign in, fill in( 'username', :with Harrison25000) click_button('login') and then check the page for something you would expect only a logged in user to see (like their username as a header).